railties/test/engine/commands_test.rb

```diff
@@ -24,7 +24,7 @@ def test_help_command_work_inside_engine
 
   def test_runner_command_work_inside_engine
     output = capture(:stdout) do
-      Dir.chdir(plugin_path) { system("bin/rails runner 'puts Rails.env'") }
+      Dir.chdir(plugin_path) { system({"SKIP_REQUIRE_WEBPACKER" => "true"}, "bin/rails runner 'puts Rails.env'") }
```

Comment:
Why we need to do this?

Reply:
@rafaelfranca The `webpacker:install` command is not run for plugins - so the webpacker.yml file doesn't exist. As a result - when you run the any command the first line in the output is `RAILS_ENV=test environment is not defined in config/webpacker.yml, falling back to production environment`. As the tests are making assertions about the output of the command the tests were failing. By not requiring `webpacker` in the gem file we don't get this warning in the output and the tests pass.